fms/models/__init__.py

```diff
@@ -207,6 +207,23 @@ def _is_dp(distributed_strategy):
     return distributed_strategy in {"fsdp", "hsdp", "ddp"}
 
 
+def _validate_unfuse_strategy(extra_args):
+    """
+    ckpt       target_model   unfuse_strategy     task
+    -------------------------------------------------------------------------------------------------------
+    fused      fused          None                copy param from ckpt into fused model
+    fused      unfused        post                copy param from ckpt into fused model, then unfuse model
+    unfused    fused          pre + fuse (?)      instantiate unfused model, copy param from ckpt into unfused model, then fuse model OR
+                                                  fuse ckpt and copy param into fused model
```

Comment:
make this comment more explicit. Say some quantization strategies require the model and checkpoint both to be the same kind of fused/unfused. For these cases, you need the "pre" strategy. Otherwise, for regular checkpoints, fusing the checkpoint and then loading into the fused model is fine